### PR TITLE
[SASS-3309] Check transfers into overseas pensions Navigation

### PR DESCRIPTION
**Pull request by adam-shahzad-2**
# Checklist

Jira subtask: [SASS-3309](https://jira.tools.tax.service.gov.uk/browse/SASS-3309)

## Developer: Before Asking Others To Review Your Code

Have you ...

- [x]  Had a QA verify that all acceptance criteria have been met, with comment(s) in the ticket? 
- [x]  Hosted a group review of your implementation?
- [x]  Rebased against `main`? 
- [ ]  Ensured that your commit(s) contain the ticket key?
- [x]  Ensured that your branch is named after the ticket key?
- [x]  Successfully run the unit tests?
- [ ]  Successfully run the integration tests?
- [ ]  Successfully run the journey tests? If not, please add a comment explaining why it wasn't necessary.
- [ ]  Ensured that the PR job has passed.

## Reviewer: Before Reviewing

Have you ...

- [ ]  Pulled the branch down?
- [ ]  Assigned yourself to this PR?
- [ ]  Moved the Jira *subtask* to “in review”?

## Developer: After Approval

Have you ...

- [ ]  Moved the Jira *subtask* to “in pipeline”?

## Developer: After Merging

Have you ...

- [ ]  Scheduled a demo? If not, please add a comment.

### CODE REVIEW

File: app/controllers/pensions/shortServiceRefunds/ShortServicePensionsSchemeController.scala [hunk 1]
```diff
@@ -93,7 +93,11 @@ class ShortServicePensionsSchemeController @Inject()(actionsProvider: ActionsPro
       schemeReference = (if (scheme.ukRefundCharge.contains(true)) scheme.pstr else scheme.qops)
         .getOrElse(""),
       providerAddress = scheme.providerAddress.getOrElse(""),
-      countryId = scheme.alphaTwoCountryCode
+      countryId = scheme.alphaTwoCountryCode.fold{
```
Comment (abiodxn) [inline app/controllers/pensions/shortServiceRefunds/ShortServicePensionsSchemeController.scala:96]: why is this change in here ?

File: conf/messages [hunk 2]
```diff
@@ -952,6 +952,19 @@ transferIntoOverseasPensions.overseasPensionSchemeTaxTransferCharge.error.tooBig
 transferIntoOverseasPensions.overseasTransferChargesPaid.title = Did a UK pension scheme pay the transfer charge to HMRC?
 transferIntoOverseasPensions.overseasTransferChargesPaid.error.noEntry = Select yes if a UK pension scheme paid the transfer charge to HMRC
 
+#Transfer Into Overseas Pensions CYA
+transferIntoOverseasPensions.cya.heading = Check transfers into overseas pensions
+transferIntoOverseasPensions.cya.transferIntoOverseasPensions = Transfers into overseas pensions
+transferIntoOverseasPensions.cya.amountCharged = Amount charged
+transferIntoOverseasPensions.cya.noAmountCharged = charge
```
Comment (abiodxn) [inline conf/messages:959]: i thought we said we'd fix this to make it say "no charge"

**Review by abiodxn — CHANGES_REQUESTED** [review]

File: app/controllers/pensions/shortServiceRefunds/ShortServicePensionsSchemeController.scala (hunk 1 above)
Comment (adam-shahzad-2) [reply app/controllers/pensions/shortServiceRefunds/ShortServicePensionsSchemeController.scala:96]: was brought up in the group review, for edge cases

File: app/utils/CYABaseHelper.scala [hunk 3]
```diff
@@ -34,35 +34,41 @@ trait CYABaseHelper{
...
+  def displayedValue(value: BigDecimal): String = if (value == 0) "" else s"£$value"
 
-   def displayedValue(valueOpt: Option[Boolean])(implicit messages: Messages): String =
-    valueOpt.map(value => if (value) messages("common.yes") else messages("common.no")).getOrElse("")
+  def displayedValue(valueOpt: Option[Boolean], suffix: Option[String] = None)(implicit messages: Messages): String =
+    valueOpt.map(value => if (value) {
+      messages("common.yes")
+    }
+    else if (suffix.isDefined) {
+      s"${messages("common.no")} ${messages(suffix.getOrElse(""))}"
```
Comment (abiodxn) [inline app/utils/CYABaseHelper.scala:64]: This should just be the suffix i think and we shouldn't have a need for concatenating strings

**Review by abiodxn — CHANGES_REQUESTED** [review]

**Review by abiodxn — APPROVED** [review]

**Pull request merged**